.github/workflows/nightly-macos-x86_64.yml

```diff
@@ -10,6 +10,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        boost:
+            - version: 1.71.0
+            - version: 1.76.0
```

Comment:
what about with latest boost? what was the reason for 1.76? Was it related to C++14?

Reply:
Yes, it is related with C++14.

Reply:
It doesn't affect the users of the library, the reason why this build is we treat the warnings as an error at Release build. We can do code change to fix that issue but i don't think it is required because there is nothing wrong with the code, it is just a preliminary warning for developers which tells `Boost.Math` will increase minimum C++ stantard to C++14.